crypto/evp_extra/evp_extra_test.cc

```diff
@@ -2674,3 +2697,71 @@ TEST_P(PerKEMTest, KAT) {
     EXPECT_EQ(Bytes(ss_expected), Bytes(ss));
   });
 }
+
+TEST_P(PerKEMTest, KeygenSeedTest) {
+  // ---- 1. Setup phase: generate a context and a key ----
+  // Create context of KEM type.
+  bssl::UniquePtr<EVP_PKEY_CTX> ctx(EVP_PKEY_CTX_new_id(EVP_PKEY_KEM, nullptr));
+  ASSERT_TRUE(ctx);
+
+  // Setup the context with specific KEM parameters.
+  ASSERT_TRUE(EVP_PKEY_CTX_kem_set_params(ctx.get(), GetParam().nid));
+
+  // Generate a key pair.
+  EVP_PKEY *raw = nullptr;
+  ASSERT_TRUE(EVP_PKEY_keygen_init(ctx.get()));
+
+  // ---- 2. Test getting the lengths only ----
+  size_t keygen_seed_len;
+  ASSERT_TRUE(EVP_PKEY_keygen_deterministic(ctx.get(), &raw, nullptr,
+                                            &keygen_seed_len));
+  EXPECT_EQ(keygen_seed_len, GetParam().keygen_seed_len);
+
+  // ---- 3. test failure mode on a seed len too small----
```

Comment:
```suggestion
  // ---- 3. Test failure mode on a seed len too small ----
```

Reply:
fixed in [ad37a81](https://github.com/aws/aws-lc/pull/1578/commits/ad37a8166eadbd1af368220b672f7c951f142f44)